docs/distributions/overview.md

```diff
@@ -2,6 +2,7 @@
 
 This is a landing page for the distribution specific guides contained in this wiki.
 
-- [Arch Linux](https://wiki.t2linux.org/distributions/arch/installation/)
-- [Manjaro](https://wiki.t2linux.org/distributions/manjaro/home/)
-- [Ubuntu](https://wiki.t2linux.org/distributions/ubuntu/home/)
+-   [Arch Linux](https://wiki.t2linux.org/distributions/arch/installation/)
+-   [Manjaro](https://wiki.t2linux.org/distributions/manjaro/home/)
+-   [Ubuntu](https://wiki.t2linux.org/distributions/ubuntu/home/)
+-   [Fedora](https://wiki.t2linux.org/distributions/fedora/home/)
```

Comment:
This list should stay in alphabetical order

Reply:
good point. done.